plugins/json_rpc/include/golos/plugins/json_rpc/api_helper.hpp

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+// Provides PLUGIN_API_VALIDATE_ARGS macro usable in DEFINE_API methods
+// to generate arguments processing boilerplate.
+
+//Usage:
+// PLUGIN_API_VALIDATE_ARGS(
+//     (std::string, json_filename)
+// )
+//Expands to:
+// std::string json_filename;
+// auto n_args = args.args->size();
+// GOLOS_ASSERT(n_args == 1, golos::invalid_parameter, "Expected 1 parameter, received ${n}", ("n", n_args)("required",1));
```

Comment:
Here will be `golos::invalid_arguments_count`

Reply:
Fixed. Thanks.